style-dictionary/classic/contexts/compact-table.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { StyleDictionary } from '../../utils/interfaces';
+import { tokens as parentTokens } from '../spacing';
+import merge from 'lodash/merge';
+import { expandDensityDictionary } from '../../utils';
+const tokens: StyleDictionary.SpacingDictionary = {
+  spaceScaledXxs: '{spaceXxxs}',
+  spaceScaledXs: '{spaceXxs}',
+  spaceScaledL: '{spaceM}',
+};
```

Comment:
We should probably override all `spaceScaled*` tokens, right? So that we also properly display custom content in cells.